test/fixtures/api/responses/get-balance-sheet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "balances": [
+    {
+      "counterparty": "rKm4uWpg9tfwbVSeATv4KxDe6mpE9yPkgJ",
+      "balances": [
```

Comment:
I don't know how I'd write a schema for your first solution. I could make it a large one-dimensional array of amounts (second suggestion) but the current format would seem preferable